packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -363,6 +363,18 @@ const pentahoPlus = makeTheme((theme) => ({
     base: "6px",
   },
   components: {
+    HvBadge: {
+      classes: {
+        showCount: {
+          ...theme.typography.caption2,
```

Comment:
NEXT 5 is also `caption2`. Let's change the "baseline" instead?

Reply:
@zettca updated to change the baseline instead.